tests/collections/test_qnode_collection.py

```diff
@@ -227,6 +227,7 @@ def test_eval_autograd(self, parallel, dask_support):
         assert np.all(res == expected)
 
     @pytest.mark.autograd
+    # pylint: disable=unused-argument
```

Comment:
Should we instead be using `parallel` when calling the qnode collection?

Reply:
Possibly? I don't know enough to make the appropriate functional change. My goal was to fix linting issues while preserving functionality as is. But since we are touching it, might make sense to address these types of issues. I would need help making an appropriate change though. 

I did check and we do use "TODO" comments in some spots in PL. Best practice is to tag those with an issue to solve. But regardless, would a TODO with an issue be sufficient here? Or would it make sense to actually fix it now?